### PR TITLE
feat(plasma-icons): Add other icons for `24 dp` and  `16 dp`

### PR DESCRIPTION
**Pull request by neretin-trike**
<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.59.0-canary.9.2273937613.0
  npm install @salutejs/plasma-icons@1.77.0-canary.9.2273937613.0
  npm install @salutejs/plasma-temple@1.61.0-canary.9.2273937613.0
  npm install @salutejs/plasma-ui@1.97.0-canary.9.2273937613.0
  npm install @salutejs/plasma-web@1.94.0-canary.9.2273937613.0
  # or 
  yarn add @salutejs/plasma-b2c@1.59.0-canary.9.2273937613.0
  yarn add @salutejs/plasma-icons@1.77.0-canary.9.2273937613.0
  yarn add @salutejs/plasma-temple@1.61.0-canary.9.2273937613.0
  yarn add @salutejs/plasma-ui@1.97.0-canary.9.2273937613.0
  yarn add @salutejs/plasma-web@1.94.0-canary.9.2273937613.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-9/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-9/
  ui storybook: http://plasma.sberdevices.ru/pr-9/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-9/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-9/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-9/temple-storybook/

File: packages/plasma-icons/src/scalable/Icon.assets.16/Battery100.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import { IconProps } from '../IconRoot';
+
+export const Battery100: React.FC<IconProps> = (props) => (
+    <svg width="100%" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+        <rect x="6" y="5" width="4" height="7" rx="0.5" fill="currentColor" />
+        <path
+            d="M4 3H6V2H10V3H12V14H4V3Z"
+            stroke="currentColor"
+            strokeOpacity="0.96"
```
Comment (Yeti-or) [inline packages/plasma-icons/src/scalable/Icon.assets.16/Battery100.tsx:11]: same

File: packages/plasma-icons/src/scalable/Icon.assets.16/Battery20.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { IconProps } from '../IconRoot';
+
+export const Battery20: React.FC<IconProps> = (props) => (
+    <svg width="100%" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+        <path
+            d="M4 3H6V2H10V3H12V14H4V3Z"
+            stroke="currentColor"
+            strokeOpacity="0.96"
```
Comment (Yeti-or) [inline packages/plasma-icons/src/scalable/Icon.assets.16/Battery20.tsx:10]: same

File: packages/plasma-icons/src/scalable/Icon.assets.16/Battery40.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { IconProps } from '../IconRoot';
+
+export const Battery40: React.FC<IconProps> = (props) => (
+    <svg width="100%" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+        <path
+            d="M4 3H6V2H10V3H12V14H4V3Z"
+            stroke="currentColor"
+            strokeOpacity="0.96"
```
Comment (Yeti-or) [inline packages/plasma-icons/src/scalable/Icon.assets.16/Battery40.tsx:10]: same

**Review by Yeti-or — APPROVED** [review]
```
cd packages/plasma-icons
perl -p -i -e 's/strokeOpacity=\"0.96\"//g' src/**/*.tsx
```

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-9/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-9/
  ui storybook: http://plasma.sberdevices.ru/pr-9/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-9/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-9/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-9/temple-storybook/

**Pull request merged**